### ISSUE TITLE
Data Sets on Google Drive

### ISSUE THREAD

**Issue by TouqeerAhmad** [ZhaomingKong/Real_Noisy_Clean_Color_IMAGES#1]
Hi, I came across your image and video denoising data sets. It would be great if you could share the data sets through another platform e.g. Google Drive or Dropbox. 

Thanks,
Touqeer

**Comment by ZhaomingKong** [comment]
Yes!!! Thank your for your valuable suggestion, but I am in China, meaning I need VPN to use some American-based platforms, but if you would like to do me this favor, I will be very grateful!!!

Thanks, 
Zhaoming Kong
